fe/fe-core/src/main/java/com/starrocks/alter/SchemaChangeJobV2.java

```diff
@@ -179,6 +179,8 @@ public class SchemaChangeJobV2 extends AlterJobV2 {
     private List<Integer> sortKeyIdxes;
     @SerializedName(value = "sortKeyUniqueIds")
     private List<Integer> sortKeyUniqueIds;
+    @SerializedName(value = "disableReplicatedStorageForGIN")
+    private boolean disableReplicatedStorageForGIN = false;
```

Comment:
This is just a check, which can be determined by olapTable.enableReplicatedStorage(). Why add a field that needs to be persisted?